src/main/resources/cassandra.conf

```diff
@@ -2,4 +2,6 @@ cassandra {
   contact-points = ["127.0.0.1:9042"]
   keyspace = "recommender"
   ratings-table-name = "ratings"
+  user-features-table-name = "modelUserFeatures"
+  product-features-table-name = "modelProductFeatures"
```

Comment:
It is easier to have all the configuration keys in one place. If there would be situations when one component load CassandraClient class but does not have those keys in the loaded configuration files the logic would be much more complicated. We would have to use Option types or hardcode some default values inside Scala files. 

I decided it is easier to just move all table name configuration inside single file.